src/locales/en/common.json

```diff
@@ -601,6 +601,7 @@
     "emptyState": {
       "title": "No crypto assets yet?",
       "desc": "Make sure the <1><0>{{managerAppName}}</0></1> app is installed and start receiving",
+        "descToken": "Make sure the <1><0>{{managerAppName}}</0></1> app is installed and start receiving <3><0>{{currencyTicker}}</0></3> and <5>ERC20 tokens</5>",
```

Comment:
ERC20 tokens needs to be generalised to get the token type, which would come from `tokenType.toUpperCase()`